tools/rimage/config/ptl.toml.h

```diff
@@ -0,0 +1,125 @@
+#include "platform-ptl.toml"
+
+	[[module.entry]]
+	name = "BRNGUP"
+	uuid = "2B79E4F3-4675-F649-89DF-3BC194A91AEB"
+	affinity_mask = "0x1"
+	instance_count = "1"
+	domain_types = "0"
+	load_type = "0"
+	module_type = "0"
+	auto_start = "0"
+
+	index = __COUNTER__
+
+	[[module.entry]]
+	name = "BASEFW"
+	uuid = "0E398C32-5ADE-BA4B-93B1-C50432280EE4"
+	affinity_mask = "3"
+	instance_count = "1"
+	domain_types = "0"
+	load_type = "0"
+	module_type = "0"
+	auto_start = "0"
+
+	index = __COUNTER__
+
+#ifdef CONFIG_COMP_MIXIN_MIXOUT
+#include <audio/mixin_mixout/mixin_mixout.toml>
```

Comment:
In some modules like mixin_mixout mod_cfg is defined for each platform:

```
#if CONFIG_METEORLAKE
	mod_cfg = [ ... ]
#elif CONFIG_LUNARLAKE
	mod_cfg = [ ... ]
#endif
```

this will result in incomplete toml for PTL so we need to also update module toml files